mava/configs/env/connector.yaml

```diff
@@ -0,0 +1,9 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+  - scenario: con-10x10x5a
```

Comment:
Can we please have a comment here with all possible scenarios configs like in other envs?